Detectors/TOF/simulation/src/Digitizer.cxx

```diff
@@ -490,7 +497,16 @@ Float_t Digitizer::getEffZ(Float_t z)
 
 //______________________________________________________________________
 Float_t Digitizer::getFractionOfCharge(Float_t x, Float_t z) { return 1; }
+//______________________________________________________________________
+void Digitizer::newTF()
```

Comment:
In the end you don't introduce the TFs on the digitizer level, right? Why do you need this here?

Reply:
I implemented and tested it before of our discussion. I decided to leave it as an option in case in future we will need it for some reasons. If it is not a problem I would keep this method.

Reply:
As you want, but the digitization should only produce a detector response.

Reply:
Ok. In the meanwhile I can move the method to the mother class (WindowFiller) so that it is no longer associated to digitization.

Reply:
DONE (visible soon in the PR)